requirements.txt

```diff
@@ -3,12 +3,12 @@ scikit-image~=0.19.2
 scikit-learn==0.24.1
 numpy~=1.20.3
 scipy==1.10.0
-matplotlib==3.3.4
+matplotlib==3.7.2
 dill==0.3.3
 pytest~=7.1.3
 importlib-resources
 typing-extensions
 
 requests~=2.22.0
 Pillow~=8.4.0
-setuptools~=49.2.1
```

Comment:
I don't know what happened there, I used the GitHub editor to change the relevant line and suddenly this appears to be changed. I think GitHub editor deleted an invisible char there